Microsoft.Build.Targets 0.1.0-preview-00041-160927 doesn't contain Microsoft.CSharp.CrossTargeting.targets

https://github.com/Microsoft/msbuild/pull/1105 brought in new files: 

- Microsoft.CSharp.CrossTargeting.targets
- Microsoft.VisualBasic.CrossTargeting.targets

And other targets files try importing these files.

However, these files are not included in the nupkg.  Therefore, when we try using this build from the CLI, it fails:

```
F:\core-sdk\.dotnet_cli\sdk\1.0.0-preview3-003701\Microsoft.CSharp.targets(168,4): error MSB4019: The imported project "F:\core-sdk\.dotnet_cli\sdk\1.0.0-preview3-003701\Microsoft.CSharp.CrossTargeting.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [F:\core-sdk\bin\Debug\Tests\PreserveCompilationContext\CompilationContext\TestApp\TestApp.csproj] [F:\core-sdk\build\build.proj] 
```

We need these new files included in the MSBuild NuGet packages.

/cc @rainersigwald @nguerrera

--- a/src/nuget/Microsoft.Build.Targets.nuspec
+++ b/src/nuget/Microsoft.Build.Targets.nuspec
@@ -31,6 +31,8 @@
   <files>
     <file src="_._" target="lib\netstandard1.3" />
     <file src="Microsoft.Common.CrossTargeting.targets" target="runtimes\any\native" />
+    <file src="Microsoft.CSharp.CrossTargeting.targets" target="runtimes\any\native" />
+    <file src="Microsoft.VisualBasic.CrossTargeting.targets" target="runtimes\any\native" />
     <file src="Microsoft.Common.CurrentVersion.targets" target="runtimes\any\native" />
     <file src="Microsoft.Common.targets" target="runtimes\any\native" />
     <file src="Microsoft.CSharp.CurrentVersion.targets" target="runtimes\any\native" />

